tests/otel_integrations/test_anthropic.py

```diff
@@ -385,7 +377,7 @@ def test_sync_messages_stream(instrumented_client: anthropic.Anthropic, exporter
                     'logfire.span_type': 'log',
                     'logfire.tags': ('LLM',),
                     'duration': 1.0,
-                    'response_data': '{"combined_chunk_content":"The answer is secret","chunk_count":3}',
+                    'response_data': '{"combined_chunk_content":"The answer is secret","chunk_count":2}',
```

Comment:
The more liberal checking in `content_from_messages` started counting an additional chunk with empty text. So I started excluding all chunks with empty text, and it turns out there was already one being counted.